core/lib/src/request/request_impl.rs

```diff
@@ -741,9 +741,8 @@ impl<'r> Request<'r> {
     pub fn query_value<'a, T>(&'a self, name: &str) -> Option<form::Result<'a, T>>
         where T: FromForm<'a>
     {
-        if self.query_fields().find(|f| f.name == name).is_none() {
-            return None;
-        }
+        // No need to continue if the name is not found.
+        self.query_fields().find(|f| f.name == name)?;
```

Comment:
I don't know...I'd prefer the obvious version above, even if they are identical.